src/main/java/core/basesyntax/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Map;
```

Comment:
Consider arranging import statements in a consistent order. For example, you could group them by the Java standard library imports, third-party library imports, and your project package imports. It's not a critical issue, but it helps in maintaining readability as the project grows.